src/main.jsx

```diff
@@ -1,13 +1,32 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import { TodosProvider } from "./store/todos-context";
-import App from './App'
-import './index.css'
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { TodosProvider } from './store/todos-context';
+// import App from './App'
+import './index.css';
+import SignUp from './components/SignUp';
+import Login from './components/LogIn';
+import ForgotPassword from './components/ForgotPassword';
 
+const router = createBrowserRouter([
+    {
+        path: '/',
+        element: <SignUp />,
+    },
+    {
+        path: 'login',
+        element: <Login />,
+    },
+    {
+        path: 'forgot-password',
+        element: <ForgotPassword />,
+    },
+]);
```

Comment:
It would be nice to move this route declaration into a separate as it might grow into many routes in the future

Reply:
Also, I think we need an error page as well. https://salehmubashar.com/blog/404-page-with-react-router-v64
https://reactrouter.com/en/main/route/error-element